pages/bookshelf/index.vue

```diff
@@ -27,6 +27,9 @@
         </div>
       </div>
     </div>
+    <div v-if="loading" class="absolute top-0 left-0 w-full h-full flex items-center justify-center">
+      <ui-loading-indicator text="Loading Library...">
```

Comment:
I just noticed when I ran this that there was an error, it didn't have a closing />
You can run your updates locally first by just doing `npm run dev` and accessing from the browser.

Reply:
Whoops! I missed that! Huh, I built that branch with `npm run generate` and `npx cap sync` to run the app on my iOS device, no errors. But I do see it with `npm run dev`, in the future I'll check that too. Sorry!

![Read_Btn](https://user-images.githubusercontent.com/62854267/164943673-f432f051-8033-42b8-ab91-ead90b5698a1.jpeg)